<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGitCloneRepositoryDialog</class>
 <widget class="QDialog" name="QGitCloneRepositoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clone Git repository</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0,0,0,0" columnstretch="0,1">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_recievingObjects">
     <property name="text">
      <string>Recieving objects</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QProgressBar" name="progressBar_recievingObjects">
     <property name="maximum">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_recievingIndexes">
     <property name="text">
      <string>Recieving indexes</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QProgressBar" name="progressBar_recievingIndexes">
     <property name="maximum">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item>
      <widget class="QLabel" name="label_status"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QProgressBar" name="progressBar_indexingDeltas">
     <property name="maximum">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_indexingDeltas">
     <property name="text">
      <string>Indexing deltas</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_creatingFiles">
     <property name="text">
      <string>Creating files</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QProgressBar" name="progressBar_creatingFiles">
     <property name="maximum">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
